src/app.js

```diff
@@ -49,6 +54,42 @@ app.get('/filmer/:movieId', async (req, res) => {
     }
   }
 });
+app.get('/api/movies/:movieId', (req, res) => {
+  const { movieId } = req.params;
+  res.render('reviews', { movieId });
+});
+
+app.post('/api/movies/:movieId/reviews', (req, res) => {
```

Comment:
Is there a way you could put most of the content of this app.post into a separate js file, where most of the functions and operations happen? I think that is the way we have done with the other routes to keep the app.js a bit cleaner and more easily overlooked. Maybe you can put it in the reviewBody.js that you already import?